webplugin/js/app/kommunicate-ui.js

```diff
@@ -1378,4 +1378,39 @@ KommunicateUI = {
             },
         });
     },
+    getUrlFromBlobKey: function(blobKey, callback) {
+        var params= '?key='+blobKey;
+        window.Applozic.ALApiService.ajax({
+            type: 'GET',
+            global: false,
+            url: MCK_BASE_URL + '/rest/ws/file/url' + params,
+            success: function (res) {
+                callback(null, res);
+            },
+            error: function (err) {
+                callback(err);
+            },
+        })
+    },
+    isInView: function (element, targetElement) {
+        const rect = element.getBoundingClientRect();
+        const targetRect = targetElement.getBoundingClientRect();
+        return (
+            rect.top >= targetRect.top &&
+            rect.left >= targetRect.left &&
+            rect.bottom <= targetRect.bottom &&
+            rect.right <= targetRect.right
+        );
+    },
+    processLazyImage: function(img){
+        img.classList.remove('lazy_image');
+        var thumbnailBlobKey = img.getAttribute('data-thumbnailBlobKey');
+        KommunicateUI.getUrlFromBlobKey(thumbnailBlobKey, function(err, thumbnailUrl){
+            if (err) throw err;
+            thumbnailUrl && (img.src = thumbnailUrl);
+            setTimeout(function(){
+                img.classList.add('lazy_image');
+            },15*60*1000);
```

Comment:
replace it with constant

Reply:
constant is not added, is it marked as resolved by mistake, or changes are not yet pushed to repo?